package.json

```diff
@@ -23,13 +24,15 @@
   "keywords": [
     "react-native-localize",
     "react-native",
+    "react-native-windows",
     "localize",
     "localization",
     "l20n"
   ],
   "scripts": {
     "format": "prettier '**/*.{js,json,md,ts,tsx}' --write",
-    "prepare": "bob build"
+    "prepare": "bob build",
+    "windows": "react-native run-windows"
```

Comment:
Shouldn't this be in the `/example` app?

Reply:
It does sound reasonable, I think I'll just remove this.  I believe this was auto added by some script.  I never use these helper script on windows, since VS has such a much better flow without it.